build.sbt

```diff
@@ -57,14 +58,18 @@ libraryDependencies ++= {
     "com.typesafe.akka" %% "akka-http-spray-json" % akkaHttpV,
     "com.typesafe.akka" %% "akka-http-testkit" % akkaHttpV,
     "com.iheart" %% "ficus" % "1.4.0",
+    "com.softwaremill.sttp" %% "core" % "0.0.16",
+    "com.softwaremill.sttp" %% "async-http-client-backend-future" % "0.0.16",
+    "com.typesafe.scala-logging" %% "scala-logging" % "3.6.0",
     // WARNING: Updating this swagger-ui version?
     // Do a search/replace for the version string ("2.1.1") in the code too or the swagger page won't be found.
     "org.webjars" % "swagger-ui" % "2.1.1",
     "org.scalatest" %% "scalatest" % scalaTestV % Test,
     "io.swagger" % "swagger-parser" % "1.0.22" % Test,
     "org.yaml" % "snakeyaml" % "1.17" % Test,
     "org.broadinstitute" %% "cromwell-api-client" % cromwellV,
-    "org.broadinstitute" %% "lenthall" % lenthallV
+    "org.broadinstitute" %% "lenthall" % lenthallV,
```

Comment:
old timey separate repos! and what is this 'lenthall' of which you speak?

Reply:
I didn't want to switch until that stuff was in master